### PR TITLE
Add more details to the running steps.

### PR DESCRIPTION
**Pull request by Mohammad-nassar10**
Signed-off-by: mohammad-nassar10 <mohammad.nassar@ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,22 +54,38 @@ kubectl apply -f fybrikapplication.yaml
...
+
+while [[ $(kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.assetStates.fybrik-notebook-sample/iceberg-dataset.conditions[?(@.type == "Ready")].status}') != "True" ]]; do echo "waiting for fybrik-notebook-sample/iceberg-dataset asset" && sleep 5; done
 ```
 
 Wait For the pod `my-notebook-default-trino-module-xxxx` to be completed. This pod runs a python code that registers the asset in trino and applies the policy to create a virtual dataset. The user can use the following username to connect to trino:
 
     "name": "user1"
 
-For example, you can run trino docker container and run queries. Run the following command to run trino server.
+For example, you can run trino docker container and run queries. First, check the docker container name of trino (the docker container with the image `trinodb/trino:latest`). Then, Run the following command to run trino server.
```
Comment (roytman) [inline README.md:67]: add `docker ps | grep trinodb/trino:latest`

File: README.md [hunk 2]
```diff
@@ -46,22 +54,38 @@ kubectl apply -f fybrikapplication.yaml
...
 show tables from iceberg.icebergtrino;
 ```
 
-You can run a query to select from the created view. I should return only allowed columns according to the policies.
+You can run a query to select from the created view. It should return only allowed columns according to the policies.
 ```bash
 select * from iceberg.icebergtrino.view1;
-```
\ No newline at end of file
+```
```
Comment (roytman) [inline README.md:80]: Please add that  in the output we don't see the columns with PII data (a, d)

we can add that if we login into the trino as admin, we can see both table "logs" and view view1`

**Review by roytman — COMMENTED** [review]
where do we install policies, and define that columns "a" and "d" are PII? We should explicitly mention it

**Comment by Mohammad-nassar10** [comment]
@roytman The policy is in `sample_assets/sample-policy.rego` and the tags of the columns is in `sample_assets/asset-iceberg`.

File: README.md [hunk 3]
```diff
@@ -46,22 +56,56 @@ kubectl apply -f fybrikapplication.yaml
 Run the following command to wait until the fybrikapplication be ready.
```
Comment (roytman) [inline README.md]: following command**s**

File: README.md [hunk 4]
```diff
@@ -1,18 +1,26 @@
...
+
+- Helm 3.3 or greater must be installed and configured on your machine.
+- Kubectl 1.18 or newer must be installed on your machine.
+- Access to a Kubernetes cluster such as Kind as a cluster administrator.
+
 ### Install trino
 Install trino and minio using the following commands.
 ```bash
-cd iceberg/trino-iceberg-minio/
+cd trino-iceberg-minio/
```
Comment (roytman) [inline README.md:13]: maybe move `cd ..` from L 23 to here

**Review by roytman — APPROVED** [review]

**Pull request merged**